guvnor-m2repo-editor/guvnor-m2repo-editor-api/src/main/java/org/guvnor/m2repo/service/M2RepoService.java

```diff
@@ -25,8 +25,6 @@
 @Remote
 public interface M2RepoService {
 
-    String getJarName( String path );
-
     String loadPOMStringFromJar( String path );
```

Comment:
this method doesn't support .pom files, it just throws the following when I click the "open" button of ArtifactListViewImpl:
ERROR 08-06 07:34:03,115 (GuvnorM2Repository.java:loadPOMFromJarInternal:634)     error in opening zip file